bindings/nodejs/index.d.ts

```diff
@@ -292,18 +292,57 @@ export class Operator {
    * Get a presigned request for read.
    *
    * Unit of expires is seconds.
+   *
+   * ### Example
```

Comment:
This example seems over designed. We only need to include `op.presignRead(path, parseInt(expires))` part.

How about the following:

```js
const req = op.presignRead(path, parseInt(expires));

console.log("method: ", req.xxx)
console.log("url: ", req.xxx)
console.log("headers: ", req.xxx)
```

Reply:
Make sense. It's actually enough.